components/fieldgroup/index.css

```diff
@@ -10,40 +10,78 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+
+/* fieldgroup/index.css
+ *
+ * fieldgroup contains four component dependences:
+ * radio, checkbox, helptext, fieldlabel
+ *
+ */
+
+/* custom properties */
 .spectrum-FieldGroup {
-  --spectrum-fieldgroup-margin: var(--spectrum-global-dimension-size-200);
-  --spectrum-fieldgroup-readonly-delineator: '\002c';
+  --spectrum-fieldgroup-margin: var(--spectrum-spacing-300);
+  --spectrum-fieldgroup-readonly-delimiter: '\002c';
 }
 
+/* field group */
 .spectrum-FieldGroup {
   display: flex;
-  vertical-align: top;
+  align-items: top;
   flex-wrap: wrap;
+
+  /* field group label top aligned */
+  &--toplabel {
+    flex-direction: column;
+  }
+
+  /* field group label side aligned */
+  &--sidelabel {
+    flex-direction: row;
+  }
 }
 
+/* input fields layout */
+.spectrum-FieldGroupInputLayout {
+  display: flex;
+  align-items: top;
+  flex-wrap: wrap;
+
+  /* input fields stacked vertically */
+  &--vertical {
+    flex-direction: column;
+  }
+
+  /* input fields aligned horizontally */
+  &--horizontal {
+    flex-direction: row;
+
+    /* space between horizontal fields */
+    .spectrum-FieldGroup-item:not(:last-child) {
+      margin-inline-end: var(--spectrum-fieldgroup-margin);
+    }
+
+    /* move help text down to new row */
+    .spectrum-HelpText {
+      flex-basis: 100%;
+    }
+  }
+}
+
+/* read-only checkbox group */
 .spectrum-FieldGroup {
   .spectrum-Checkbox.is-readOnly {
     .spectrum-Checkbox-box {
       display: none;
     }
 
+    /* read-only checkbox fields delimited by commas */
     &:not(:last-child) {
       .spectrum-Checkbox-label {
         &:after {
-          content: var(--spectrum-fieldgroup-readonly-delineator)
+          content: var(--spectrum-fieldgroup-readonly-delimiter)
```

Comment:
ah, now I see it. lol. Subtle difference between a delineator and delimiter. 
<img width="920" alt="Screen Shot 2022-08-09 at 22 04 48" src="https://user-images.githubusercontent.com/52184321/183811014-fcaaa911-3596-4a71-b3ab-fb696e80b8a6.png">